InSource/Helpers/Constants.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Sdl.Community.InSource.Helpers
+{
+	public class Constants
+	{
+		public readonly string NotificationGroupId = "b0261aa3-b6a5-4f69-8f94-3713784ce8ef";
+		public readonly string CreateProjectText = "Create project";
```

Comment:
Move this into resource file

Reply:
Done. the CreateProjectText and CreateNewProject text were moved to PluginResources.resx file